src/components/MultipleAvatars.js

```diff
@@ -70,7 +70,7 @@ const MultipleAvatars = (props) => {
     if (props.icons.length === 1 && !props.shouldStackHorizontally) {
         return (
             <View style={avatarContainerStyles}>
-                <Tooltip text={props.avatarTooltips[0]}>
+                <Tooltip text={props.shouldShowTooltipText && props.icons[0].name}>
```

Comment:
This will cause text to be a boolean. It's supposed to be a string.

Let's do this instead:
```js
const tooltipTexts = shouldShowTooltipText ? _.pluck(icons, 'name') : [];
```

and use tooltipTexts the same way avatarTooltips was used.

Reply:
I think we can revert to `shouldShowTooltip`.